docs/documentation/development/bindings/categories.md

```diff
@@ -14,7 +14,44 @@ The thing type definition allows to specify a category. User interfaces can pars
 
 | Category        | Description                                          |
 |-----------------|------------------------------------------------------|
-
+| Aircondition    | Air condition devices incl. Fans |
+| BinarySensor    | General purpose sensor that returns a boolean value, i.e. a contact that reports open/close on a door or window |
+| Bridge          | Bridges/Gateway need to access other devices like used by Philips Hue for example |
+| Boiler          | Devices that heat up water |
+| Camera          | All kinds of cameras |
+| Car             | Smart Cars |
+| Cistern         | Cistern |
+| Clock           | Devices that provide the current time |
+| Coffee          | Coffee machines |
+| Fridge          | Fridges, Freezers |
+| Furniture       | Smart furniture |
+| GarageDoor      | Garage doors |
+| GeneralSensor   | General purpose sensor that does not fit in any category |
+| Heating         | Devices that deal with heating |
+| Hifi            | Devices related to TV/audio, such as satelite or cable receivers, recorders, etc.
+| HomeApplicance  | Devices that look like Waschingmachines, Dishwashers, Dryers, etc. |
```

Comment:
This is far too broad - a home appliance is almost everything.
As we have "Fridge" as a separate category, we should probably do the same for the rest here -or we combine it as "white good".